EcalTiming/plugins/EcalTimingCalibProducer.cc

```diff
@@ -330,19 +429,50 @@ EcalTimingCalibProducer::Status EcalTimingCalibProducer::duringLoop(const edm::E
 	edm::Handle<EERecHitCollection> eeRecHitHandle;
 	iEvent.getByLabel(_ecalRecHitsEETAG, eeRecHitHandle);
 
+	std::cout << "[DEBUG]" << "\t" << ebRecHitHandle->size() << "\t" << eeRecHitHandle->size() << std::endl;
 
+   
+	eventTimeMap_.clear();
+	
+	timeEB  = EcalCrystalTimingCalibration();
+	timeEEM = EcalCrystalTimingCalibration();
+	timeEEP = EcalCrystalTimingCalibration();
+    
 	// loop over the recHits
 	// recHit_itr is of type: edm::Handle<EcalRecHitCollection>::const_iterator
-	std::cout << "[DEBUG]" << "\t" << ebRecHitHandle->size() << std::endl;
 	for(auto  recHit_itr = ebRecHitHandle->begin(); recHit_itr != ebRecHitHandle->end(); ++recHit_itr) {
 		addRecHit(*recHit_itr); // add the recHit to the list of recHits used for calibration (with the relative information)
 	}
 
 	// same for EE
 	for(auto recHit_itr = eeRecHitHandle->begin(); recHit_itr != eeRecHitHandle->end(); ++recHit_itr) {
-		addRecHit(*recHit_itr);
+		addRecHit(*recHit_itr); // add the recHit to the list of recHits used for calibration (with the relative information)
 	}
 
+	// If we got less than the minimum recHits, continue
+	if(eventTimeMap_.size() < _recHitMin) return kContinue;
```

Comment:
Since this is configurable in the python, could we just set it to 0 when we
don't want it?

Otherwise how would you suggest setting isSplash? through config or by
looking at the events?

-Peter

On Fri, Apr 24, 2015 at 1:16 AM, shervin86 <notifications@github.com> wrote:

> In EcalTiming/plugins/EcalTimingCalibProducer.cc
> <https://github.com/shervin86/CalibCalorimetry/pull/2#discussion_r29025854>
> :
>
> >  	}
> >
> > +	// If we got less than the minimum recHits, continue
> > +	if(eventTimeMap_.size() < _recHitMin) return kContinue;
>
> this is valid only for splash events, need an if(isSplash) maybe?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/shervin86/CalibCalorimetry/pull/2/files#r29025854>.
>